packages/integrations/markdoc/README.md

```diff
@@ -344,6 +344,41 @@ Now, you can call this function from any Markdoc content entry:
 
 📚 [See the Markdoc documentation](https://markdoc.dev/docs/functions#creating-a-custom-function) for more on using variables or functions in your content.
 
+### Markdoc Language Server
+
+If you are using VS Code, there is a [Markdoc language extension](https://marketplace.visualstudio.com/items?itemName=Stripe.markdoc-language-support). Which includes syntax highlighting and autocomplete for configured tags.
+For more information, [see the language server on GitHub](https://github.com/markdoc/language-server.git).
+
+To set up the extension, create a `markdoc.config.json` file into the project root, with following content:
+
+```json
+[
+  {
+    "id": "my-site",
+    "path": "src/content",
+    "schema": {
+      "path": "markdoc.config.mjs",
+      "type": "esm",
+      "property": "default",
+      "watch": true
+    },
+    "routing": {
+      "frontmatter": "route"
+    }
+  }
+]
+```
+
+The `schema` property, is the most important. It contains all information to configure the language server.
+It accepts following properties.
+
+- `path`: The path to the configuration file.
+- `type`: Kind of module your configuration file uses.
+- `property`: The exported property name that contains the configuration object.
+- `watch`: Tell the server to watch for changes in the configuration.
+
+The global `path` property, tells the server where the content is located. You can't use a glob pattern. 
```

Comment:
```suggestion
The `schema` property contains all information to configure the language server for Astro content collections.
It accepts following properties:

- `path`: The path to the configuration file.
- `type`: The type of module your configuration file uses (`esm` allows `import` syntax).
- `property`: The exported property name that contains the configuration object.
- `watch`: Tell the server to watch for changes in the configuration.

The top-level `path` property tells the server where content is located. Since Markdoc is specific to content collections, you can use `src/content`.
```

Correcting some improper comma usage and sentence fragments. I also rephrased the "global `path`" section to use "top-level `path`", since I was unsure what global meant at first.